### DISCUSSION TITLE
Automatic translate optional API params

### DISCUSSION THREAD

**Discussion by devjamin** [WeblateOrg/weblate#12730] [Q&A] [answered]
Hi you all,

We're using Weblate to translate our website content, newsletters etc. and we often have parts of strings like brand names that shouldn't be translated. Here we're adding `<span translate="no">Brand Name</span>` to tell our translators to not translate that bit. We're automatically translating it via DeepL to speed up translators time as they then have a foundation and only need to proof-read. Problem lies in that DeepL automatic translate DOES translate these things encapsulated in notranslate spans, however they actually support blocking this by adding query param `?tag_handling=html`. Unfortunately I can only seem to find where to set the base api url and nowhere to specify additional params for deepl api

**Comment by nijel** [comment, marked as answer]
Have you tried adding the brand name to Weblate glossary as non translatable term? Glossaries are synced to DeepL, I'm just not sure if non translatable terms are handled as well.

**Reply by devjamin** [reply]
Thanks for the answer!
I added some non-translatable terms to glossary and DeepL indeed now doesn't automatically translate them, hurray :) Guess i'll just have to make a sync job syncing brand names and other non-translateable terms to glossary as they often change - but it will work and that's great.
